app/views/templates/MainTemplate.scala.html

```diff
@@ -57,7 +57,7 @@
     HmrcStandardPageParams(
         serviceURLs = ServiceURLs(
             serviceUrl = Some(routes.IndexController.start.url),
-            signOutUrl = Some(routes.SignedOutController.signedOut.url)
+            signOutUrl = Some(routes.SignedOutController.signOutToExitSurvey.url)
```

Comment:
Here you've changed the behaviour to redirect to feedback survey, while previously it wasn't doing it.